marketplace/config.json

```
@@ -1,12 +1,470 @@
-[{
-  "id": "demos",
-  "external": true,
-  "title":"Demos",
-  "logo": "https://2715296027-files.gitbook.io/~/files/v0/b/gitbook-x-prod.appspot.com/o/spaces%2FxZnkmJ00s3wgxLckYeeU%2Fuploads%2FVfSH1jvjvoVvgckTL3J5%2FDemosLogo1.2.svg?alt=media",
-  "shortDescription": "Enhanced Web3 security with intuitive and robust Sybil resistance solutions.",
-  "categories": ["Dev Tooling", "Data & Analytics"],
-  "author": "Demos",
-  "url": "https://app.demos.gloabl",
-  "description": "Enhanced Web3 security with intuitive and robust Sybil resistance solutions.",
-  "site": "https://www.demos.global"
-}]
\ No newline at end of file
+[
+  {
+    "id": "desyn",
+    "external": true,
+    "title": "DeSyn",
+    "logo": "/assets/desyn.png",
```

@rmilrad the logos have to be prefixed with the full GitHub path. For instance, this must be:

```suggestion
    "logo": "https://raw.githubusercontent.com/hemilabs/blockscout-config/refs/heads/master/assets/desyn.png",
```